guides/source/testing.md

```diff
@@ -859,292 +846,440 @@ users(:david, :steve)
 Model Testing
 -------------
 
-Model tests are used to test the various models of your application.
+Model tests are used to test the various models of your application and their
+associated logic. You can test this logic using the assertions and fixtures that
+we've explored in the sections above.
 
-Rails model tests are stored under the `test/models` directory. Rails provides
-a generator to create a model test skeleton for you.
+Rails model tests are stored under the `test/models` directory. Rails provides a
+generator to create a model test skeleton for you.
 
 ```bash
 $ bin/rails generate test_unit:model article title:string body:text
 create  test/models/article_test.rb
-create  test/fixtures/articles.yml
 ```
 
-Model tests don't have their own superclass like `ActionMailer::TestCase`. Instead, they inherit from [`ActiveSupport::TestCase`](https://api.rubyonrails.org/classes/ActiveSupport/TestCase.html).
+Model tests don't have their own superclass like `ActionMailer::TestCase`.
```

Comment:
Maybe it would be helpful to show an example model test file, that the generator creates?